src/style.css

```diff
@@ -0,0 +1,113 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+} */
+
+/* html {
+  font-family: "Avenir", Arial, sans-serif;
+} */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-weight: 300px;
+}
+
+.form-big {
+  position: relative;
+  min-width: 200vh;
+  margin-top: 20px;
+  margin-right: 8px;
+  margin-left: 8px;
+  background-position: center;
+}
+
+.img-big {
+  position: absolute;
+  padding-left: 26px;
+  top: 25px;
+  bottom: 26px;
```

Comment:
it's redundant

Reply:
if I remove it, it will be like in the photo.
![Знімок екрана 2022-12-25 102007](https://user-images.githubusercontent.com/119611480/209461395-be4a6dfc-e3ab-4195-b243-c2aac1483e3a.png)